include/net/sntp.h

```diff
@@ -82,6 +82,20 @@ int sntp_query(struct sntp_ctx *ctx, u32_t timeout,
  */
 void sntp_close(struct sntp_ctx *ctx);
 
+/**
+ * @brief Convenience function to query SNTP in one-shot fashion
+ *
+ * Convenience wrapper which calls getaddrinfo(), sntp_init(),
+ * sntp_request(), and sntp_close().
+ *
+ * @param server Address of server in format addr[:port]
+ * @param timeout Query timeout
+ * @param time Timestamp including integer and fractional seconds since
+ * 1 Jan 1970 (output).
+ */
```

Comment:
It'd be good to document the return value of this function.

Reply:
> It'd be good to document the return value of this function.

Fixed, thanks.